ch.bailu.gtk_meteo.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app-id": "ch.bailu.gtk_meteo",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "gtk-meteo.sh",
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk17/install.sh"
+      ]
+    },
+    {
+      "name": "gtk-meteo",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/bailuk/gtk-meteo.git",
+          "branch": "543eb0a5ff40b57d70277fb6c7ac9b3e6f9fb431"
```

Comment:
NEVER branch. This is a commit btw.

Reply:
Changed to commit and updated upstream (see below)